moped-database/metadata/tables.yaml

```diff
@@ -824,6 +824,32 @@
       - status_id
       filter: {}
       check: {}
+  event_triggers:
```

Comment:
Changes in this file bring new tables into the activity log system and also makes /all/ fields in the tracked tables tracked fields. We were missing a few here and there.